zabbix_auto_config/processing.py

```diff
@@ -393,6 +397,8 @@ def disable_host(self, zabbix_host):
                 logging.info("Disabling host: '%s' (%s)", zabbix_host["host"], zabbix_host["hostid"])
             except pyzabbix.ZabbixAPIException as e:
                 logging.error("Error when disabling host '%s' (%s): %s", zabbix_host["host"], zabbix_host["hostid"], e.args)
+            except IndexError:
+                logging.error("Host group 'All-auto-disabled-hosts' not found, please create it.")
```

Comment:
I feel that it is quite severe if the group is missing. Currently the application will crash, which I think is kind of "OK".

If you catch the exception nothing bad will happen other than a log event.

If it shouldn't crash maybe at least raise a `ZACException`?

And maybe keep logging the hostname that couldn't be disabled, like in the above except?

Reply:
I dropped this commit for now, these groups are very important and should be handled in another way if at all.